### DISCUSSION TITLE
Python Setuptools

### DISCUSSION THREAD

**Discussion by spudgun1001** [hnimminh/libresbc#172] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/7355b5d9-2c2e-46f2-9dca-b50d86db4e34)

I have recently encountered this issue when running the playbook.

I have ran the following command:

   sudo apt install python3-pip

To install Python dependencies.

Am I missing something?

Very sorry for the constant questions

**Comment by spudgun1001** [comment]
I have also checked, setuptools is already under "/usr/lib/python3/dist-packages"

**Comment by spudgun1001** [comment]
Fixed that issue...now I have this...

![image](https://github.com/user-attachments/assets/23948a32-5cbc-47de-96fe-b37c31d554ab)

Already ran "sudo apt update" and ran "sudo apt autoremove" to remove any broken packages, but no luck

**Reply by hnimminh** [reply]
@spudgun1001 
suppose that with ansible, you only need to run `ansible-playbook` command and everything should work. if you got the issue it mean: 
- you're doing incorrect (configuration.. )
- or dependency change..

I would ask many question because of missing information:
- what is your debian version?  - above error It seems SignalWise change something on dependency for `buster`, can you retry with `bullsyes` (buster is EOL already)
- what is your ansible version?
- is your  ansible_user is sudoer?

**Reply by spudgun1001** [reply]
I have changed my debian to bullsyes however I now get this issue.

![image](https://github.com/user-attachments/assets/cb123dd4-09df-4221-96cb-60f2a6531061)


My ansible user is a sudoer, My Ansible is on core 2.17.3

**Reply by hnimminh** [reply, marked as answer]
From the error, it seems you're typo `bullseye` not `bullsyes` 🙂
